templates/flavors/rke2/vlan/kustomization.yaml

```diff
@@ -0,0 +1,148 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
```

Comment:
full includes the etcd backups, autoscaling etc are all part of the full template - do we not want it?

Reply:
No, this kustomization should only include patches needed to run the default flavor with VLAN support instead of VPCs (might want to base off vpcless instead)

Reply:
VPCless has a very different CiliumPolicy (no direct routing etc) - could build on default though